docker-compose.yaml

```diff
@@ -22,6 +22,8 @@ services:
       HASURA_GRAPHQL_UNAUTHORIZED_ROLE: public
       HASURA_GRAPHQL_LOG_LEVEL: debug
       HASURA_GRAPHQL_ENABLE_CONSOLE: 'true'
+      HASURA_GRAPHQL_EXPERIMENTAL_FEATURES: ${HASURA_GRAPHQL_EXPERIMENTAL_FEATURES}
```

Comment:
Could you please fall back to some defaults here? This should probably be added to the example docker compose file as well.